R/error-checks.R

```diff
@@ -52,3 +56,7 @@ validateIsSameLength <- function(...) {
 
   stop(messages$errorDifferentLength(arguments))
 }
+
+validateNotAllNull <- function(...){

```

Comment:
A todo here. I am not sure if we need this. I want to make sure that at least one of the arguments is not null

Reply:
Can you give an example?

Reply:
I have a method that takes two optional arguments, but at least one of them should not be NULL.
I will change the signature of my method anyways so this won't be required but it could be useful...maybe ?

Reply:
Yepp, could be useful.